test/e2e/singlecluster/pod_test.go

```diff
@@ -349,8 +381,8 @@ var _ = ginkgo.Describe("Pod groups", func() {
 						gomega.Expect(ok).To(gomega.BeTrue())
 						event, ok := evt.Object.(*v1.Event)
 						gomega.Expect(ok).To(gomega.BeTrue())
-						objKey := types.NamespacedName{Namespace: event.InvolvedObject.Namespace, Name: event.InvolvedObject.Name}
-						if defaultGroupPods.Has(objKey) && event.Reason == "Stopped" {
+						if event.InvolvedObject.Namespace == ns.Name && event.Reason == "Stopped" {
```

Comment:
This is not needed strictly for the PR, but it is related. I makes the checked condition stronger - we verify there were no other pods preempted in the namespace.